### PR TITLE
Add plus-icon for AppNavigationHeaderNewEvent

### LINKED ISSUE #3945: "+ New event", the + should be an icon and not in text

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
fixes #3945

### CODE REVIEW

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 1]
```diff
@@ -26,13 +26,18 @@
 		class="button primary new-event"
 		@click="newEvent"
 		@shortkey="newEvent">
-		{{ $t('calendar', '+ New event') }}
+		<Plus :size="20" decorative />
```
Comment (tcitworld) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:29]: The `decorative` prop [has been removed](https://github.com/robcresswell/vue-material-design-icons/releases/tag/5.0.0) in vue-material-design-icons 5.0.0.

**Review by tcitworld — CHANGES_REQUESTED** [review]

**Review by tcitworld — APPROVED** [review]

**Comment by ChristophWurst** [comment]
>  ERROR in src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue
>  54:4  ✖  Expected indentation of 2 tabs  indentation
>  55:4  ✖  Expected indentation of 2 tabs  indentation

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 2]
```diff
@@ -26,13 +26,18 @@
...
+		{{ $t('calendar', 'New event') }}
 	</button>
 </template>
 
 <script>
+import Plus from 'vue-material-design-icons/Plus.vue'
 export default {
 	name: 'AppNavigationHeaderNewEvent',
+	components: {
+	  Plus,
```
Comment (ChristophWurst) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:39]:
```suggestion
		Plus,
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Comment by ChristophWurst** [comment]
The build still fails

```
ERROR in src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue
 54:4  ✖  Expected indentation of 2 tabs  indentation
 55:4  ✖  Expected indentation of 2 tabs  indentation
```

File: src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue [hunk 3]
```diff
@@ -43,3 +48,10 @@ export default {
 	},
 }
 </script>
+
+<style scoped>
+	.button.primary.new-event {
+			display: flex;
+			align-items: center;
```
Comment (ChristophWurst) [inline src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue:55]:
```suggestion
		display: flex;
		align-items: center;
```

**Pull request merged**